lib/services/remove_product_from_basket.rb

```diff
@@ -0,0 +1,11 @@
+module Store
+  class RemoveProductFromBasket
+    def initialize(item_id)
+      @item_id = item_id.to_i
+    end
+
+    def call
+      BASKET.delete_if { |item| item.id == @item_id }
```

Comment:
it should check quantity probably, if you have more then one in basket, it should remove one